demo/index.html

```diff
@@ -30,17 +30,17 @@
                     "marginCountryPopupForLeft" : 60
                 },
                 'colors': {
-                    "hover" : "red",
-                    "highlighted": "yellow"
+                    "hovercoloroption" : "orange",
+                    "highlightedcoloroption": "yellow"
```

Comment:
Actually, I think the first naming is much better. It's completely clear if you read
```
colors: { hover: ..., highlighted: ... }
```